apps/portals/src/configurations/elportal/synapseConfigs/projects.ts

```diff
@@ -106,7 +106,7 @@ const projects: SynapseConfig = {
       'name',
       'institutions',
       'principalInvestigators',
-      'grantNumber',
+      'grant',
```

Comment:
Plot `'grant'` on the projects explore page: 
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/e6a2edca-382c-4841-b48d-fbf3f9056bb1)